pontoon/base/locales.py

```diff
@@ -0,0 +1,130 @@
+"""
```

Comment:
The problem is that we need those codes fields of FilterEntities. Every instance of FilterEntities contains a set of fields for every locale. I couldn't find a reasonable way to build one model from the `Locale.objects.all()`.